src/main/java/API_hw2/constants/BoardConstant.java

```diff
@@ -0,0 +1,19 @@
+package API_hw2.constants;
```

Comment:
you keep constants and enums together and call the package constants. What about altering the package name? Or storing them separately?

Reply:
renamed package